source/language.js

```diff
@@ -25,13 +34,8 @@ Language = Space.messaging.Serializable.extend('Language', {
 // Register as EJSON type
 Language.type('Language');
```

Comment:
Thanks for the catch, this is a bug with the current implementation. The auto-type-registration currently only works with nested namespaces. Since these types are top-level it does not register correctly.